R/plot.R

```diff
@@ -140,3 +150,36 @@ plot_gam_splines <- function(gam_model, tract, df_tract, dwi_metric, covariates,
     device = "png"
   )
 }
+
+
+#' Plot profiles for each bundle and each metric.
+#'
+#' @param df Tibble object
+#' @param groups 
+#' @param group_names 
+#' @param group_by
+#' @param group_by_name
+#' @param bins 
+#' @param quantiles
+#' @param palette
+#'
+#' @export
+plot_tract_profiles <- function (
```

Comment:
We want this function to do something reasonable per default and then have many options to tweak. 

So, if it just gets a df as input, it would create one panel per bundle, one figure per metric of all of the bundles and metrics in that dataframe. But, you can also provide optional arguments that let you choose metrics and bundles (as character vectors, that's the right term, right?). 

Then, another optional argument would let you group in each panel by another column of the df. If that column is not a factor, you group into n (n=3? n=5? should also be an optional argument) quintiles and have a separate line for each quintile group.